docs/general/networking/nginx.md

```diff
@@ -388,3 +391,102 @@ error_page 460 http://your-page-telling-your-limit/;
 ```
 
 [See here for more](https://www.nginx.com/blog/rate-limiting-nginx/)
+
+## Nginx Proxy Manager
+
+[Nginx Proxy Manager](https://nginxproxymanager.com/) (pronounced "engine X") was created by jc21 in 2016 and provides an easy-to-use web GUI for Nginx.
```

Comment:
Pronounciation is wrong as it skips the "proxy manager" part. I also don't think it's worth mentioning that part. Just tell us what the tool is, a jellyfin documentation reader doesn't care who made it and how they pronounce it.